internal/pkg/model/relation.go

```diff
@@ -48,6 +51,39 @@ type Relation interface {
 
 type Relations []Relation
 
+func (v Relations) Equal(v2 Relations) bool {
```

Comment:
Trocha som sa zasekal pri rieseni `diff`.
Potrebujem vediet, ci su 2 `Relations` (local/remote) rovnake, a ak nie, tak aky je v nich rozdiel.

Chcel som to riesit cez oficialny balicek `cmp`, ale asi je tam nejaky bug,  zacykli sa to (bezdovodne):
![image](https://user-images.githubusercontent.com/19371734/140355329-1543bb1d-e912-431f-899c-d72eeb8234c6.png)


Preto som implementoval "rucne" `Relations.Equal` a `Relations.Diff`.